src/components/molecules/RecipePreviewCard/RecipePreviewCard.tsx

```diff
@@ -1,26 +1,30 @@
 import Link from "next/link"
 
+import { FavoriteCount } from "../../atoms/FavoriteCount/FavoriteCount"
 import { RecipePreviewPicture } from "../../atoms/RecipePreviewPicture/RecipePreviewPicture"
 import { RecipePreviewText } from "../../atoms/RecipePreviewText/RecipePreviewText"
 
-interface RecipePreviewCardProps {
+type RecipePreviewCardProps = {
   recipe: {
     index: number
     url: string
     title: string
     content: string
+    count: number
   }
 }
 
 export const RecipePreviewCard = ({ recipe }: RecipePreviewCardProps) => {
   return (
-    <div className="grid gap-2 justify-center w-[173px] h-[214]">
+    <div className="grid gap-2 justify-center w-[194px] h-[248px]">
       <Link href="">
+        <div className="relative">
+          <FavoriteCount count={recipe.count}></FavoriteCount>
+        </div>
         <RecipePreviewPicture index={recipe.index} url={recipe.url}></RecipePreviewPicture>
       </Link>
       <RecipePreviewText previewTitle={recipe.title} previewContent={recipe.content}></RecipePreviewText>
     </div>
```

Comment:
こんな感じに書くとシンプルかなと思います！

```suggestion
    <Link href={recipe.linkUrl} className="grid gap-2 justify-center w-[173px] h-[214px] relative">
      <RecipePreviewPicture index={recipe.index} url={recipe.imagePass} />
      <RecipePreviewText previewTitle={recipe.title} previewContent={recipe.content} />
      <FavoriteCount count={recipe.count} />
    </Link>
```

Reply:
確かに！
そのようにしました👍